android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
```

Comment:
OK then, use it, but don't delete DB_VERSION and add a comment to lock db schema once next gen API is complete